superduperdb/backends/base/query.py

```diff
@@ -242,18 +243,32 @@ def __eq__(self, other):
             parts=self.parts + [('__eq__', (other,), {})],
         )
 
-    def __leq__(self, other):
+    def __lt__(self, other):
         return type(self)(
             db=self.db,
             identifier=self.identifier,
-            parts=self.parts + [('__leq__', (other,), {})],
+            parts=self.parts + [('__lt__', (other,), {})],
         )
 
-    def __geq__(self, other):
+    def __gt__(self, other):
         return type(self)(
             db=self.db,
             identifier=self.identifier,
-            parts=self.parts + [('__geq__', other)],
+            parts=self.parts + [('__gt__', (other,), {})],
+        )
+
+    def __le__(self, other):
```

Comment:
I think we will have issues parsing `t.select(t.brand == 'Nike')`, for example.

Reply:
I tried it, it worked.